src/deployment/deployment-role.json

```diff
@@ -3,6 +3,7 @@
     "Description": "Permissions required for OneFuzz deployment",
     "Actions": [
         "Microsoft.Authorization/locks/*",
+        "Microsoft.Keyvault/vaults/*",
```

Comment:
Is there someplace where we should add/update documentation about this new permission?

Reply:
At the moment, no.

When we integration keyvault support, we'll need to specifically call out in the release notes.  (Hence the breaking-change label)